docs/UserGuide.md

```diff
@@ -214,7 +214,7 @@ Example:
 
 Adds an issue to the housing management system.
 
-Format: `iadd r/ROOM_NO d/DESCRIPTION [t/TIMESTAMP] [s/STATUS] [c/CATEGORY]`
+Format: `iadd r/ROOM_NO d/DESCRIPTION [t/TIMESTAMP] [s/STATUS] [c/CATEGORY] [g/TAG]`
```

Comment:
I didn't know how to use multiple tags at first. Can consider putting command example for both:
1. single tag 
`iadd r/10-100 d/Broken light c/Furniture g/ HIGH
2.  multiple tag
`iadd r/10-100 d/Broken light c/Furniture g/ HIGH g/ URGENT